app/models/endpoint.rb

```diff
@@ -6,6 +6,10 @@ class Endpoint < ApplicationRecord
   belongs_to :endpoint_type
   has_and_belongs_to_many :preservation_policies
 
+  enum delivery_class: {
```

Comment:
add a comment saying:
` # NOTE: DO not change the underlying contents for enum values that have been merged to master?`

just as a reminder to anyone viewing this code?
what do you think?

Reply:
yeah, and we did it w/ the status enum's so just trying to stay consistent without commenting

Reply:
I would delete the comments in the other location if consistency is required.  I dislike the other comment because it seems like notes scribbled in the margin of a textbook example.  IMO, the merged-to-master-ness of the `enum` isn't something that needs to be warned about... particularly in code that is *already* merged to master.  Code should never have to know what branch it is on.  Also, the branch 100% doesn't matter: the blocker is persisted **data** attached to the enum that would require migration.

The scenario of a maintainer familiar w/ specifying only the keys is one where there is *even less* freedom to modify the code, since the order of elements must be maintained.  Not sure why that person would now feel inclined to change integer values.  (We don't put similar warnings in our migration files or the schema definition for the table, though the constraints are even stronger there.)

I can add a link to the docs page for active-record enums, if that suffices.  It is appropriate to document dependencies in the app and point to clarifying docs.  But to the extent that we are using common dependencies in a highly conventional manner, IMO, it is mildly inappropriate for our comments to assume the role of educating others (or ourselves!) how those common dependencies themselves work in conventional application.  That isn't directly part of the app and we aren't really the authority on it anyway!

Reply:
fair points.  i like your wording better than what i'd left on the other enum, so thanks for changing it there too 🙂.  that comment was indeed conflating a merge to master with the existence of data that needs migration.

anyway, i pretty much entirely agree, other than that i don't feel bad occasionally stating what might be obvious or conventional -- in this case, i'm just imagining a rails neophyte or someone who's been away for a while not thinking through the consequences of changing the mapping.  maybe that's exceptionally unlikely, making this comment noise, but my gut feeling was that this was an appropriate place for some over-explanation.